File: app/src/main/java/jy/cn/com/yframework/simple/db/TestDao.java
package jy.cn.com.yframework.simple.db;

import jy.cn.com.ylibrary.db.Scope;

/**
 * @Author Administrator
 * @Date 2019/10/28-13:59
 * @TODO
 */
public class TestDao {
    @Scope(isPrimaryKey = true, isAutoKey = true)
    public int id;
    public String savePath;
    public int countLength;
    public int readLength;
    public int connectionTime;
    public int stateInt;
    @Scope(isCompareField = true)
    public String url;
    public int updateProgress;
}

File: app/src/main/java/jy/cn/com/yframework/simple/db/TestDao.kt
package jy.cn.com.yframework.simple.db

import jy.cn.com.yframework.simple.db.bean.TestInfo


/**
 * @Author Administrator
 * @Date 2019/10/28-13:59
 * @TODO
 */
object TestDao {

    private var dbWrapper: DBWrapper<TestInfo>? = null

    init {
        dbWrapper = DBWrapper(TestInfo::class.java)
    }

    fun getInstance(): DBWrapper<TestInfo> {
        return dbWrapper!!
    }
}
